kosat-core/src/commonMain/kotlin/org/kosat/CDCL.kt

```diff
@@ -416,7 +416,14 @@ class CDCL {
 
     // ---- CDCL functions ---- //
 
-    // add new constraint and watchers to it, executes only in newClause
+    /**
+     * Add new constraint and watchers to it.
```

Comment:
Since we eliminated the word "constraint" from the interface (method names), also do eliminate it from the comments.